Added German translation (de)

Acer54 opened the pull request:
I have added german translation for all files in def/de. Tested with Atom 1.14.2 x64.
![screen_atom_de](https://cloud.githubusercontent.com/assets/17161393/23106567/ba1d569e-f6ef-11e6-999e-af6d987295f7.png)

liuderchi commented on def/de/menu_win32.cson:
```diff
@@ -32,270 +32,270 @@ Menu:
-      "&Copy":
-        value: "Copy(&C)"
+        value: "Wiederhe&rstellen"
+      "&Cut":
```
Considering [point #1 of CONTRIBUTING.md](https://github.com/liuderchi/atom-i18n/blob/master/CONTRIBUTING.md#translation-guide), please keep the key unchanged for both *Line57* and *Line59*

[ ] Line 57
[ ] Line 59

Acer54 replied:
Ah, I see. this was a copy / paste Error from the menu_linux.cson. Will fix this today later on.

liuderchi commented on def/de/menu_linux.cson:
```diff
@@ -141,161 +141,161 @@ Menu:
         submenu:
           "Open In &Dev Mode…":
-            value: "Open In Dev Mode(&D)..."
+            value: "Öffnen im %Dev Modus..."
```
Is `%Dev` a part of translation?

Acer54 replied:
This was mainly because of the hotkey issue... I thought it would be better to stick to the abbreviation "Dev" = Developer = "Entwickler" (german) ... but I think "Dev" is also a known abbreviation in Germany, but I will exchange this in "Öffnen im Entwicklungs-Modus (&D)"

liuderchi commented on def/de/menu_linux.cson:
```diff
@@ -1,133 +1,133 @@
 Menu:
   "&File":
-    value: "File(&F)"
+    value: "Datei"
```
Considering [point #3 of CONTRIBUTING.md](https://github.com/liuderchi/atom-i18n/blob/master/CONTRIBUTING.md#translation-guide), we can preserve hotkey hint by appending `(&F)` to enhance UX.

Any other idea is welcome to discuss : )

Acer54 replied:
You are right, I have tried to find characters inside the german words, matching the correct hotkey... but If I didnt find one, I have missed to integrate the correct hotkey in brackets. Will rework this.

liuderchi commented:
this PR is about #29

Acer54 commented:
Hi Liuderchi, I've updated the hotkey-Issue (which looks quite unusual...) and also fixed the mistakes in the key-definition in win32.cson file. Regards Matthias

liuderchi commented:
@Acer54 Currently hot key hint is NOT always appearing in the end wrapped with braces.

I want to confirm that this kind of inconsistency is acceptable for you.

IMO it's OK to merge now but I just want to make sure. :+1:

Acer54 commented:
@liuderchi I've never used hotkeys like this, because of this it just looks unusual for me ... but anyway, it might be helpful for others which used to work with hotkeys. Do I have to do something else, or is my job done now :-) Dont know exactelly... never made a pull request until now. Regards.

liuderchi commented:
@Acer54 Thanks for your contribution and it's now my job to merge it into master.

Because basically it's an irreversable operation so it was just an confirmation.
